assets/webviews/styles/welcome.css

```diff
@@ -10,6 +10,12 @@ h2 {
   margin-bottom: 12px;
```

Comment:
This adds a margin below the title text, but since `.title-wrapper` now adds that margin, you can remove it here